core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -3884,7 +3884,7 @@ public void _testWinPartClause() {
 
     winSql(
         "select cume_dist() over w , ^rank()^\n"
-        + "from emp \n"
+        + "from emp group by deptno\n"
```

Comment:
Adding the GROUP BY to this query preserves the intent of the query.  However, we can add more testcase(s) to verify issues (I mentioned them in the JIRA)